genomesubsampler/parseRepophlan.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017--, genome-subsampler development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+#
+# parser for RepoPhlAn-downloaded genomes
+#
+
+import click
+import pandas as pd
+
+
+def parse_repophlan(repophlan_wscores_fp):
+    """ Compute basic statistics of RepoPhlAn-downloaded genomes
+
+    Parameters
+    ----------
+    repophlan_wscores_fp: string
+        file path to RepoPhlAn summary table with scores
+
+    Returns
+    -------
+    list of string
+        Human-readable report of basic statistics of genomes
+    """
+    df = pd.read_table(repophlan_wscores_fp, index_col=0, header=0)
+    out = []
```

Comment:
instead of using a list I would prefer a dict, where the key is the "human readable explanation" and the values are the numbers/strings you gather from the dataframe. It will then be easier to use your results, because res[1] is less intuitive than res['Number of RefSeq genomes']